QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -329,7 +331,14 @@ public StartResult Start(bool waitForExit)
                     {
                         TraceIbLauncherLogFile();
 
-                        _lastStartResult = new StartResult(ErrorCode.InitializationTimeout);
+                        var additionalMessage = string.Empty;
+                        if (_isFirstStart && _isAuthenticating)
+                        {
+                            // unable to complete logon because IB weekend server reset is in progress
+                            additionalMessage = "The logon process could not be completed because the IB server is busy or being reset for the weekend, please try again later.";
```

Comment:
Should `_isFirstStart` be better set to `false` here? there is a return a few lines bellow, if we go through it `_isFirstStart` would be `true` but not right. `_isAuthenticating` could be volatile maybe, since it's set by another thread?